src/oscar/templates/oscar/catalogue/reviews/review_form.html

```diff
@@ -16,7 +16,7 @@
                     </label>
                     <div class="controls">
                         {{ form.score }}
-                        <div class="star-rating {% if form.score.value == '1' %}One{% elif form.score.value == '2' %}Two{% elif form.score.value == '3' %}Three{% elif form.score.value == '4' %}Four{% elif form.score.value == '5' %}Five{% endif %}">
+                        <div class="star-rating {{ form.score.value|as_stars }}">
```

Comment:
You need to load `reviews_tags` at the top of this file, otherwise the filter cannot be found

Reply:
indeed - thanks for spotting this as well